OrbitGl/App.cpp

```diff
@@ -1034,11 +1035,46 @@ DataView* OrbitApp::GetOrCreateDataView(DataViewType type) {
     case DataViewType::kAll:
       FATAL("DataViewType::kAll should not be used with the factory.");
 
+    case DataViewType::kTracepoints:
+      if (!tracepoints_data_view_) {
+        tracepoints_data_view_ = std::make_unique<TracepointsDataView>();
+
+        if (!options_.grpc_server_address.empty()) {
+          grpc::ChannelArguments channel_arguments;
+
+          channel_arguments.SetMaxReceiveMessageSize(std::numeric_limits<int32_t>::max());
+          grpc_channel_ = grpc::CreateCustomChannel(
+              options_.grpc_server_address, grpc::InsecureChannelCredentials(), channel_arguments);
```

Comment:
I think the channel is alredy constructed - you do not need to do this again here.

Reply:
At this point, the grpc_channel_ was null so that is why I thought about constructing it.